### PR TITLE
V1.2.0 Authenticator Sample App was added

### PR DESCRIPTION
**Pull request by amitnadirping**
V1.2.0 Authenticator Sample App was added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
-Authenticator Sample App
+# Authenticator sample app
+
+This sample application demonstrates an authenticator-only application that uses PingOne for Customers mobile SDK. An iOS developer can easily build a branded and customized Authenticator application using this sample.
+
+## What’s in the sample app?
+
+  - Full native application written in Swift 5.1, compatible for all iOS devices from iOS 10 or above.
+  - Integration with PingOne SDK version 1.2.0.
```
Comment (asafd1) [inline README.md]: `Integration with PingOne SDK version 1.2.0.` --> `Integration with PingOne for Customers mobile SDK.`
also this should be a link to the sdk (on **master** branch).

File: README.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+
+## Prerequisites
+
+The Authenticator sample app requires Xcode 11 and above to compile and run.
+
+To set up your application for  working with push messages in iOS refer to [PingOne mobile SDK iOS readMe].
+
+## Installation
+
+1. Download the latest code [here].
```
Comment (asafd1) [inline README.md]: shouldn't this say:
`Download the latest code from **this** repo`?

File: README.md [hunk 3]
```diff
@@ -1 +1,44 @@
...
+    - App Icon: `AppIcon`
+    ##### Note: 
+    It is mandatory to replace these images before submitting the app to AppStore, in order to create a unique app complying with Apple's restrictions. For more information, refer to [App Store Review Guidelines].
+
+4. If needed, update the `Localizable.strings` file, to customize any string in the app.
+5. Build and run the project.
+##### Note: For further understanding the code implementation of this app, refer to [Setup a mobile app] using the PingOne SDK sample code.
+
+
+[here]: <https://github.com/pingidentity/pingone-customers-mobile-sdk-ios/tree/v1.2.0>
```
Comment (asafd1) [inline README.md]: use **master** branch

File: README.md (hunk 1 above)
Comment (amitnadirping) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (amitnadirping) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (amitnadirping) [reply README.md]: Fixed

**Review by segev-ping — APPROVED** [review]

**Pull request closed without merging**